src/main/app/claims/models/claim.ts

```diff
@@ -58,6 +59,7 @@ export class Claim {
   moneyReceivedOn: Moment
   reDetermination: ReDetermination
   reDeterminationRequestedAt: Moment
+  template: Object
```

Comment:
Can we please avoid using a generic Object here?

Reply:
sure

Reply:
Removed the generic Object and created an interface for the template